### ISSUE TITLE
AWS Provider Alpha Release

### ISSUE THREAD

**Issue by muvaf** [crossplane-contrib/provider-tf-aws#23]
Make all CRDs in provider-tf-aws use new configuration toggles to implement all XRM features.

**Comment by jbw976** [comment]
Let's also do a little bug bash with Upbounders that are outside of the purple squad.  I'd be happy to take them for a test spin too.  Some thoughts:

* it would be nice to have a specific time to do this when people trying them out for the first time can get support from the purple-squad for any issues they run into
* a bit of structure here would encourage more participation, especially folks with busy schedules.
* it doesn't need to be an rigorously organized full on bug bash, but a little mini one where folks have all the info and pointers they need to try it out, try to break it, and report issues
* this should be done before the alpha release, to get our confidence/quality up

**Comment by turkenh** [comment]
Closed with https://github.com/crossplane-contrib/provider-tf-aws/releases/tag/v0.2.0
